### DISCUSSION TITLE
how to get partition info from an asset that triggered an asset sensor

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18286] [Q&A]
do you have any good example if I want to run a job that sends the data from an asset partition to an API
basically when an asset_sensor triggers to an partitioned asset, I want to run this job that will send the partition data to an api

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1700852750375519?thread_ts=1700852750.375519&cid=C01U954MEER).*

**Comment by clairelin135** [comment]
Looks like this is a duplicate of https://github.com/dagster-io/dagster/discussions/18257
